class/topolvm.yml

```diff
@@ -1,5 +1,11 @@
 parameters:
   kapitan:
+    dependencies:
+      - type: git
+        source: https://github.com/topolvm/topolvm
+        ref: ${topolvm:charts:topolvm}
+        subdir: charts/topolvm
+        output_path: dependencies/topolvm/helmcharts/topolvm/
```

Comment:
Adding the version into the path prevents an old cache is used.

Reply:
See https://github.com/projectsyn/component-keycloak/blob/master/class/keycloak.yml#L8

Reply:
Ah, makes sense.